src/tests/editor.test.js

```diff
@@ -780,22 +780,22 @@ describe('Editor', () => {
                 await testEditor(BasicEditor, {
                     contentBefore: '<h1>]<b>abcd</b></h1><p>ef[gh</p>',
                     stepFunction: deleteForward,
-                    contentAfter: '<h1>[]gh</h1>',
+                    contentAfter: '<h1><b>[]</b>gh</h1>',
                 });
             });
             it('should not break unbreakables', async () => {
                 await testEditor(BasicEditor, {
                     contentBefore:
                         '<table><tbody><tr><td>a[bc</td><td>de]f</td></tr></tbody></table>',
                     stepFunction: deleteForward,
-                    contentAfter: '<table><tbody><tr><td>a[]</td><td>f</td></tr></tbody></table>',
+                    contentAfter: '<table><tbody><tr><td>a[bc</td><td>de]f</td></tr></tbody></table>',
```

Comment:
Wait what ? No, no.

Reply:
We discussed this on Discord before and had come to the conclusion that unbreakables rollbacking all delete actions would be an acceptable tradeoff. I don't think it would be too hard to put it back the way it was before. Just a little bit more code and since I believed it to be an acceptable tradeoff I didn't add that code, as per usual direction.